tutorials/multiple_robots.md

```diff
@@ -168,7 +192,7 @@ To set the robot namespace, SSH into your TurtleBot 4 and run the turtlebot4 set
 turtlebot4-setup
 ```
 
-Navigate to 'Bash Setup' in the 'ROS Setup' menu, then change the `ROBOT_NAMESPACE` setting. 
+Navigate to 'Bash Setup' in the 'ROS Setup' menu, then change the `ROBOT_NAMESPACE` setting.
```

Comment:
Superfluous new line or maybe this should be two paragraphs (flagged because the space was removed as part of that other discussion)